<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>268</width>
     <height>295</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>组播地址</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="AdressEdit"/>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QPushButton" name="JoinBtn">
      <property name="text">
       <string>加入组播</string>
      </property>
      <property name="iconSize">
       <size>
        <width>17</width>
        <height>6</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="UnjoinBtn">
      <property name="text">
       <string>退出组播</string>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="SendBtn">
      <property name="text">
       <string>发送消息：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="SendEdit"/>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QTextEdit" name="SendText"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>绑定端口号</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="3">
     <widget class="QLineEdit" name="PortEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
